styleguide/components/31-molecules/frequently-visited/_frequently-visited.scss

```diff
@@ -1,6 +1,12 @@
 .frequently-visited {
```

Comment:
Same here... Isn't this quite specific for Stad Gent?

Reply:
This is a styleguide component (added in #549)